db/migrations/0003_groupprofileaddress.up.sql

```diff
@@ -0,0 +1,39 @@
+create table profiles
+(
+  id serial primary key,
+  account_id text,
+  name text,
+  notes text
+);
+
+create table groups
+(
+	id serial primary key,
+	account_id text,
+	name text,
+	notes text,
+	profile_id int REFERENCES profiles(id)
+);
+
+
+create table group_contacts
+(
+	group_id int REFERENCES groups(id) ON DELETE CASCADE,
+	contact_id int REFERENCES contacts(id) ON DELETE CASCADE,
+	primary key (group_id, contact_id)
+);
+
+create table addresses
+(
+	account_id text,
+	address text,
+	city text,
+	country text,
+	home_address_contact_id bigint,
```

Comment:
This should be a foreign key pointing to contact id, and probably of `int` type according to contact schema.

Reply:
And probably cascaded delete should be enabled as well.